components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -1002,11 +1042,19 @@ static void psa_key_management_operation(void)
                 SPM_PANIC("SPM read length mismatch");
             }
 
+            partition_id = psa_identity(msg.handle);
+
             switch (psa_key_mng.func) {
                 case PSA_GET_KEY_LIFETIME: {
                     size_t lifetime_length = msg.out_size[0];
                     psa_key_lifetime_t lifetime;
 
+                    if (!psa_crypto_access_control_is_handle_permitted(psa_key_mng.handle,
```

Comment:
please see comment above

Reply:
so we'll have a huge if here? `if PSA_GET_KEY_LIFETIME || PSA_SET_KEY_POLICY || PSA_IMPORT_KEY || PSA_GET_KEY_INFORMATION || ETC...` please see above answer :)